LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/FunctionBundler/FunctionBundlerPreferredGatewayExecutionItem.cs

```diff
@@ -15,8 +14,6 @@ public void Prepare(FunctionBundlerExecutionContext context, FunctionBundlerRequ
             if (request is null) throw new ArgumentNullException(nameof(request));
 
             request.FunctionItems |= FunctionBundlerItemType.PreferredGateway;
-
-            Logger.Log(context.LoRaDevice.DevEUI, "FunctionBundler ADR request finished preparing.", LogLevel.Debug);
```

Comment:
Is there a reason why this got removed?

Reply:
I moved the log statement to the call site, where we already have a logger injected.